src/core_atmosphere/mpas_atm_core.F

```diff
@@ -48,6 +48,9 @@ function atm_core_init(domain, startTimeStamp) result(ierr)
       use mpas_attlist, only : mpas_modify_att
       use mpas_string_utils, only : mpas_string_replace
       use mpas_atm_halos, only: atm_build_halo_groups, exchange_halo_group
+#ifdef MPAS_USE_MUSICA
+      use mpas_atm_chemistry, only: chemistry_init
+#endif
```

Comment:
I might be mis-remembering, but I thought we decided to use the `MPAS_USE_MUSICA` macro inside the `mpas_atm_chemistry` module. I think it would make more sense to use it to wrap calls to functions in `mpas_musica`. Then, inside functions like `chemistry_init()`, you could have something like:
```
#ifdef MPAS_USE_MUSICA
  call musica_init()
#endif
```

Reply:
Moved `#ifdef MPAS_USE_MUSICA` to module `mpas_atm_chemistry`.